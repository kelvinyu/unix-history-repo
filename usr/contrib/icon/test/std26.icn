record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("x := [array(),table(),write,main,[],&input,1,\"abc\",'aa',2.0,&null,create 1] ----> ",image(x := [array(),table(),write,main,[],&input,1,"abc",'aa',2.0,&null,create 1]) | "none")
   write("x := sort(x) ----> ",image(x := sort(x)) | "none")
   write("every write(image(!x)) ----> ",image(every write(image(!x))) | "none")
   write("!image ----> ",image(!image) | "none")
end
