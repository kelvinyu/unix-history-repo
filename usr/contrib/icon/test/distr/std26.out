x := [array(),table(),write,main,[],&input,1,"abc",'aa',2.0,&null,create 1] ----> list(12)
x := sort(x) ----> list(12)
&null
1
2.0
"abc"
'a'
co-expression(0)
&input
function write
procedure main
list(0)
table(0)
record array(7)
every write(image(!x)) ----> none
Run-time error 116 at line 12 in std26.icn
invalid type to element generator
offending value: function image
